### PR TITLE
Documentation for newly added Omnibus Nano V6 board

### PR DESCRIPTION
**Pull request by Hwurzburg**
@tridge merged this board last night...here is the documentation I prepared for it

### CODE REVIEW

File: common/source/docs/common-omnibusnanov6.rst [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+   -  Built-in OSD
+   -  There is no logging support, so disable in parameters
+
+Where to Buy
+============
+
+- available from multiple retailers including `myairbot.com <https://store.myairbot.com/flight-controller/omnibus-f3-f4/omnibusf4prov3.html>`__
+
+Versions
+==============
```
Comment (khancyr) [inline common/source/docs/common-omnibusnanov6.rst]: too much ====

File: common/source/docs/common-omnibusnanov6.rst [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+Telem is available at UART 1
+
+The shared UART3/I2C pins are enabled only for I2C operation to allow external compass or digital airspeed sensor attachment.
+
+RC input is via pad marked LED on the board and is compatible all RX serial protocols supported by Ardupilot.
+The Buzzer output pad has no functionality.
+
+Flashing Firmware
+========================
```
Comment (khancyr) [inline common/source/docs/common-omnibusnanov6.rst]: to much ====

File: common/source/docs/common-omnibusnanov6.rst [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+Enter DFU Mode
+--------------
+To do this, you'll need to locate the DFU jumper on your board. On most board flavours this is a little push button thas needs to be
+pressed while connecting your board to your PC via USB: 
+
+.. image:: ../../../images/omnibusf4_dfu_button.png
+    :target: ../_images/omnibusf4_dfu_button.png
+
+Install dfu-util
+-----------------
```
Comment (khancyr) [inline common/source/docs/common-omnibusnanov6.rst]: too much -

File: common/source/docs/common-omnibusnanov6.rst [hunk 4]
```diff
@@ -0,0 +1,140 @@
+.. _common-omnibusf4pro:
+
+==================
```
Comment (khancyr) [inline common/source/docs/common-omnibusnanov6.rst]: not enough ===

File: common/source/docs/common-omnibusnanov6.rst [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+Omnibus F4 Nano V6 (must be V6)
+==================
```
Comment (khancyr) [inline common/source/docs/common-omnibusnanov6.rst]: not enough ===

Comment (Hwurzburg) [reply common/source/docs/common-omnibusnanov6.rst]: I dont understand...looking at other board documents there does not seem to be any standard for the length of ===== or -----
in fact, most of the ones in this document were copied directly from its parent existing omnibusf4pro document and are not my creation...

Comment (khancyr) [reply common/source/docs/common-omnibusnanov6.rst]: try to compile the wiki, you will see the syntax warning. I will a commit to correct to other. Those are annonying as they flood the promp and make us miss the real issues.
I would prefer that the correction is done in this pr. But if you cannot (don't know, don't have time etc.) do it, I will do it later

Comment (Hwurzburg) [reply common/source/docs/common-omnibusnanov6.rst]: not interested in setting up to enable compiling, but did run ReText and cleaned up as much as I could...also, changed == and -- to match header lengths (ReText does not complain about those)...about as much as I can do at this point

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**